samples/c_app/README.md

```diff
@@ -52,14 +52,43 @@ $ LD_LIBRARY_PATH=$PWD/bin:$LD_LIBRARY_PATH PCIDEVICE_INTEL_COM_SRIOV=0000:01:02
 
 #### Hugepage Requests and Limits
 To test the hugepage request and limit are being provided to a container via
-via the Downward API, the values need to be provided in the associated files.
+the Downward API, the values need to be provided in the associated files.
 If the application is not actually running in a container, then the files
-can be created manually:
+can be created manually.
+
+To simulate values being set by the
+[SR-IOV Network-Resource-Injector](https://github.com/k8snetworkplumbingwg/network-resources-injector),
+include the hugepage size and container name:
+`hugepages_{1G|2M}_{request|limit}_<ContainerName>`
+
+For example:
+```
+sudo sh -c 'echo "1024" >>/etc/podnetinfo/hugepages_1G_request_sriov-example'
+sudo sh -c 'echo "1024" >>/etc/podnetinfo/hugepages_1G_limit_sriov-example'
+```
+
+SR-IOV Network-Resource-Injector also injects an environment variable into
```

Comment:
the word "also" is confusing and unnecessory

Reply:
Updated the paragraph before to more clearly indicate that NRI is injecting hugepage fields, so now the also indicates NRI also injects an env.